app/src/main/java/com/asma/tasky/core/presentation/MainActivity.kt

```diff
@@ -44,35 +45,68 @@ class MainActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
+                    val listItems = arrayOf("Event", "Task", "Reminder")
+                    var showMenu by remember {
+                        mutableStateOf(false)
+                    }
```

Comment:
I'd always put such states in a ViewModel to make them survive config changes :)